### DISCUSSION TITLE
Fixes

### DISCUSSION THREAD

**Discussion by serifonium** [aeolus-1/idlenoughts#20] [General]
1. Shift Click to Buy Max Upgrade
2. When you recycle an Ai the price goes down
3. Better 1D board
4. Better Gold Board
5. Playtime Hours Across Current Playthrough
6. New Skill (not "Basic") (new skill tree or smth)
7. More toe shop stuff?

**Comment by aeolus-1** [comment]
1. I am working on buttons to upgrade all the boards/ai (might add the shift click later(not that importent))
2. no
3. what do you mean
4. shut up it's fine
5. good idea, gonna add statistics
6. skill tree is something i want to add
7. yeah like more maximium boards and stuff, still need more ideas and stuff
